src/ApiModules/customSuggestions.ts

```diff
@@ -28,6 +28,7 @@ export interface Hotkey {
   id: string;
   trigger: string;
   text: string;
+  type: string; //generated vs custom
```

Comment:
We can improve JS type a bit more by accepting only correct values for the `type`. And it describes right away possible values you can use.
```suggestion
  type: "generated" | "custom"; // generated vs custom
```